custom_components/lghorizon/translations/nl.json

```diff
@@ -4,15 +4,15 @@
     "step": {
       "user": {
         "title": "LG Horizon - Provider en gebruikersnaam",
-        "description":"Vul de gegevens van je Ziggo, Telenet, UPC, Magenta of Virgin account in.",
+        "description": "Vul de gegevens van je Ziggo, Telenet, UPC, Magenta of Virgin account in.",
         "data": {
           "country_code": "Provider",
           "username": "Gebruikersnaam"
         }
       },
-      "credentials":{
-        "title": "LG Horizon - Credentials",
-        "description":"Vul de credentias van je Ziggo, Telenet, UPC/Sunrise, Magenta of Virgin account in.",
+      "credentials": {
+        "title": "LG Horizon - Inloggegevens",
```

Comment:
I used credentials because of the fact that the username isn't provided in this step, but probably we should name it 'Wachtwoord of refreshtoken' ?